### DISCUSSION TITLE
rename boring

### DISCUSSION THREAD

**Discussion by xi7** [tanujnotes/Olauncher#416] [Ideas]
this very boring after rename show empty box, why you need do is like edit.

![Screenshot_20231030-081406_Olauncher](https://github.com/tanujnotes/Olauncher/assets/67331785/ecda53e9-f39e-455b-9c6c-d63f4ff61050)

do u can see, i will need rewrite all name again, but i just need emoji to category Games / Tools

**Comment by autumnyard** [comment]
Agree with this one. Editing the name should begin with the actual name already written.

It is easier to remove it than to re-write it from scratch.

Edit: it is already in an issue https://github.com/tanujnotes/Olauncher/issues/406
